yoti-sdk/src/main/AndroidManifest.xml

```diff
@@ -3,7 +3,6 @@
     package="com.yoti.mobile.android.sdk">
 
     <application
-        android:allowBackup="true"
         android:label="@string/app_name"
         android:supportsRtl="true">
```

Comment:
This one can also cause problems for integrators, should remove it too

Reply:
Hi @RobertLK. Done in https://github.com/getyoti/android-sdk-button/pull/15/commits/6a557f41cb104d7fd73ab694de8171b7bcdf386e.